pom.xml

```diff
@@ -102,6 +102,8 @@
         <module>webtau-browser</module>
         <module>webtau-http</module>
         <module>webtau-http-groovy</module>
+        <module>webtau-graphql</module>
```

Comment:
I don't necessarily mind the reordering here but I'm curious what the logic is for the order we have ended up with.  It's not alphabetical nor does it seem to be grouped in any way.

Reply:
at the bottom is all sort of shaded/dist/e2e modules, almost like private functions